File: src/main/java/org/bipmed/brave/server/search/SearchResponse.java
package org.bipmed.brave.server.search;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.bipmed.brave.server.variant.Variant;

import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class SearchResponse {

    private Integer draw;
    private Long recordsTotal;
    private Long recordsFiltered;
    private String error;
    private List<Variant> data;
}

File: src/main/kotlin/org/bipmed/brave/server/search/SearchResponse.kt
package org.bipmed.brave.server.search

import org.bipmed.brave.server.variant.Variant

data class SearchResponse(
        val draw: Int? = null,
        val recordsTotal: Long? = null,
        val recordsFiltered: Long? = null,
        val error: String? = null,
        val data: List<Variant>? = null
)